config/locales/es.yml

```diff
@@ -85,6 +85,10 @@ es:
       show_dashboard:
       reset:
       save_key:
+      multifactor_auth: Autenticación de múltiples factores
```

Comment:
Please don't add locals beside en. This will require additional review from people who know this language.